config/v1/types_infrastructure.go

```diff
@@ -1340,13 +1340,14 @@ type VSpherePlatformNodeNetworking struct {
 // +kubebuilder:validation:XValidation:rule="!has(oldSelf.ingressIPs) || has(self.ingressIPs)",message="ingressIPs list is required once set"
 type VSpherePlatformSpec struct {
 	// vcenters holds the connection details for services to communicate with vCenter.
-	// Currently, only a single vCenter is supported.
+	// Currently, only a single vCenter is supported, but in tech preview 3 vCenters are supported.
```

Comment:
We should explain that they cannot be added/remove/changed once configured, so that the validation you're adding is expected